building/building-archlinux-template.md

```diff
@@ -8,77 +8,355 @@ redirect_from:
 - /wiki/BuildingArchlinuxTemplate/
 ---
 
-Template building
-=================
+## Archlinux template building instructions ##
 
-The archlinux VM is now almost working as a NetVM. Based on qubes-builder code, you could find below how to build it and problem that could arise from template building to using archlinux as a netvm:
+<br>
+**These are the instructions for Qubes 3.2. They will take you step by step through the entire process start to finish**
```

Comment:
Shouldn't this be changed to 3.1?
There are much more occurrences of "3.2" on this page. Maybe some general comment here would be useful? Not sure if the instruction as-is will work on 3.1 - probably yes (after `s/3.2/3.1/`).